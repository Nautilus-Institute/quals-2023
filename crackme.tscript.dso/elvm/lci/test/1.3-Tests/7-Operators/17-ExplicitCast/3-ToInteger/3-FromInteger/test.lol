HAI 1.3
	VISIBLE MAEK 0 A NUMBR
	VISIBLE MAEK 1 A NUMBR
	VISIBLE MAEK 123456 A NUMBR
	VISIBLE MAEK -123456 A NUMBR
KTHXBYE
