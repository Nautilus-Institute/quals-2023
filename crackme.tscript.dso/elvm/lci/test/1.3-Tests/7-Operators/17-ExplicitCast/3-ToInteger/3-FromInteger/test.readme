This test checks that the explicit cast operator correctly casts integer values
to integer types.
