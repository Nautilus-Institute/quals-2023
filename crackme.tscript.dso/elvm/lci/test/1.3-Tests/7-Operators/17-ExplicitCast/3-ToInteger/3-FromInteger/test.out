0
1
123456
-123456
